src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -130,6 +133,20 @@ class ProxyService(proxyConfig: ProxyConfig,
     }
   }
 
+  private def logRequestForMetrics(userInfo: UserInfo, request: HttpRequest, response: HttpResponse) = {
```

Comment:
I wonder if this should be a directive in the route layer instead. It's kind of similar to `logRequestResult` with our own formatting applied: https://doc.akka.io/docs/akka-http/current/routing-dsl/directives/debugging-directives/logRequestResult.html